t/ingredients_tags.t

```diff
@@ -72,6 +51,7 @@ my @tests = (
 	[ { lc => "fr", ingredients_text => "Céleri - rave, choux - fleurs, béta - carotène"}, [ "en:celeriac", "en:cauliflower", "en:e160" ], ],
 	[ { lc => "fr", ingredients_text => "Pâte de cacao de Madagascar, café"},["fr:Pâte de cacao de Madagascar", "en:coffee"]],
 	[ { lc => "es", ingredients_text => "Vinagre, chile rojo y sal."},["en:vinegar", "en:red-chili-pepper", "en:salt"]],
+	[ { lc => "fr", ingredients_text => "Farine de blé 56 g* ; beurre concentré 25 g* (soit 30 g* en beurre reconstitué); sucre 22 g* ; œufs frais 2 g"}, [ "en:wheat-flour", "en:concentrated-butter", "fr:soit 30 g en beurre reconstitué", "en:sugar", "en:fresh-egg" ], ],
```

Comment:
do you test accuracy of % somewhere ?

Reply:
What do you mean? This is just to extract the % listed in the ingredients list we have,.